histomicstk/segmentation/nuclear/detect_nuclei_kofahi.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
+import scipy as sp
+import skimage.morphology
+import histomicstk.filters.shape as htk_shape_filters
+import histomicstk as htk
+
+
+def detect_nuclei_kofahi(im_nuclei_stain, foreground_threshold, min_radius,
+                         max_radius, min_nucleus_area, local_max_search_radius):
+
+    """Performs a nuclear segmentation using kofahi's method.
```

Comment:
Add a brief description of the method here. In short, it uses scale-adaptive multi-scale Laplacian-of-Gaussian filtering for blob enhancement + local maximum clustering for segmentation. The original paper uses Laplacian of Gaussian but we replace it Difference of Gaussian to improve speed.